src/store/ChartStore.js

```diff
@@ -120,11 +120,11 @@ class ChartStore {
         if (!this.context) { return; }
 
 
-        if (this.rootNode.clientWidth > 1100) {
+        if (this.modalNode.clientWidth > 1100) {
```

Comment:
note: resize observer is now attached to `modalNode` (the smartcharts div) instead of `rootNode` (cq-context component)